README.md

```diff
@@ -83,6 +84,12 @@ The main config of the plugin which controls how different file types are conver
 
 ⚠ The **options** object is actually the same one from the [imagemin-webp](https://www.npmjs.com/package/imagemin-webp) plugin so check their documentation for the available settings.
 
+#### overrideExtension
+
```

Comment:
Please insert the type and default value like at the rest of the options:

`Type: boolean`
`Default: true`